sklearn/mixture/bayesian_mixture.py

```diff
@@ -63,30 +63,38 @@ def _log_wishart_norm(degrees_of_freedom, log_det_precisions_chol, n_features):
 
 
 class BayesianGaussianMixture(BaseMixture):
-    """Variational estimation of a Gaussian mixture.
+    """Variational Bayesian estimation of a Gaussian mixture.
 
     This class allows to infer an approximate posterior distribution over the
     parameters of a Gaussian mixture distribution. The effective number of
     components can be inferred from the data.
 
+    This class implements two different prior type for the weight distribution:
+    a finite mixture model with Dirichlet distribution and an infinite mixture
+    model with the Dirichlet Process. In practice the approximate the Dirichlet
+    Process inference algorithm uses a truncated distribution with a fixed
```

Comment:
In practice Dirichlet Process inference algorithm is approximated and uses a truncated distribution with a fixed...